docs/state.md

```diff
@@ -384,14 +384,38 @@ WindowStore
   ledgerInfo: {
     creating: boolean,
     created: boolean,
+    delayStamp: number,
     reconcileStamp: number,
-    reconcileDelay: ?,
-    delayStamp: ?,
-    transactions: Array,
+    reconcileDelay: number,
+    transactions: [ {
+      viewingId: string,
+      surveyorId: string,
+      contribution: {
+        fiat: {
+          amount: number,
+          currency: string
+        },
+        rates: {
+          currency1: number
+          ...
+        },
+        satoshis: number,
+        fee: number
+      },
+      submissionStamp: number,
+      submissionId: string,
+      count: number,
+      satoshis: number,
```

Comment:
what's the difference between this `satoshis` and the `satoshis` in `contribution` above?